README.md

```diff
@@ -77,83 +75,192 @@ This section covers all the available commands in SkyWalking CLI and their usage
+```
+
+</details>
+
+<details>
+
+<summary>Query endpoints of a specific service</summary>
+
+If you have already got the `id` of the service:
+
+```shell
+$ ./bin/swctl endpoint ls --service-id=3
+```
+
+otherwise,
+
+```shell
+./bin/swctl service ls projectC | jq '.[].id' | xargs ./bin/swctl-latest-darwin-amd64 endpoint ls --service-id 
+[{"id":"22","name":"/projectC/{value}"}]
+```
+
+</details>
+
+<details>
+
+<summary>Query a linear metrics graph for an instance</summary>
+
+If you have already got the `id` of the instance:
 
-## Compiling and building
-Clone the source code and simply run `make clean && make` in the source directory,
-this will download all necessary dependencies and build a binary file in `./bin/swctl`.
+```shell
+$ ./bin/swctl --display=graph linear-metrics --name=service_instance_resp_time --id 5
+```
+
+otherwise
 
 ```shell
-make clean && make
+$ ./bin/swctl instance ls --service-name=projectC | jq '.[] | select(.name == "projectC-pid:7895@skywalking-server-0001").id' | xargs ./bin/swctl --display=graph linear-metrics --name=service_instance_resp_time --id
 ```
 
-## Writing a new command
-All commands files locate in directory [`commands`](commands), and an individual directory for each second-level command,
-an individual `go` file for each third-level command, for example, there is a directory [`service`](commands/service) for command `swctl service`, 
-and a [`list.go`](commands/service/list.go) file for `swctl service list` command.
+</details>
 
-Determine what entity your command will operate on, and put your command `go` file into that directory, or create one if it doesn't exist,
-for example, if you want to create a command to `list` all the `instance`s of a service, create a directory `commands/instance`,
-and a `go` file `commands/instance/list.go`.
+<details>
 
-## Reusing common options
-There're some [common options](#common-options) that can be shared by multiple commands, check [`commands/flags`](commands/flags)
-to get all the shared options, and reuse them when possible, an example shares the options is [`commands/service/list.go`](commands/service/list.go#L35)
+<summary>Query a single metrics value for a specific endpoint</summary>
 
-## Running tests
-Before submitting a pull request, add some test code to test the added/modified codes,
-and run the tests locally, make sure all tests passed.
+```shell
+$ ./bin/swctl endpoint ls --service-id 3 | jq '.[] | select(.name == "Kafka/test-trace-topic/Consumer/test").id' | xargs ./bin/swctl single-metrics --name endpoint_cpm --ids
+[{"id":"3","value":116}]
+```
+
+</details>
+
+<details>
+
+<summary>Query metrics single values for all endpoints of service of id 3</summary>
 
 ```shell
-go test -v ./...
+$ ./bin/swctl endpoint ls --service-id 3 | jq '.[] | [.id] | join(",")' | xargs ./bin/swctl single-metrics --name endpoint_cpm --end='2019-12-02 2137' --ids
+[{"id":"3","value":116}]
```

Comment:
And, you should list the output here, the graph I mean.

Reply:
these two only has one value output, adding graph makes no sense, so the graph output is added above, which have multiple values